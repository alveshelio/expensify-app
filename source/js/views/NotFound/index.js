import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <div>
    <h1>NotFound</h1>
    <p>Sorry but the page you are looking for is not here</p>
    <Link to='/'>Back to Home</Link>
  </div>
);

export default NotFound;
