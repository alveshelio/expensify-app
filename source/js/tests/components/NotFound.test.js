/* eslint no-undef: "off" */
import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../../views/NotFound';

test('it should render the NotFound component', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
