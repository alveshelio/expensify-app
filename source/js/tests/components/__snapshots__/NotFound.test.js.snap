// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the NotFound component 1`] = `
<div>
  <h1>
    NotFound
  </h1>
  <p>
    Sorry but the page you are looking for is not here
  </p>
  <Link
    replace={false}
    to="/"
  >
    Back to Home
  </Link>
</div>
`;
